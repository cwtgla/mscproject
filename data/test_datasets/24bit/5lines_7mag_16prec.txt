127.14914914
15.22223399
0.19442424
-102.4241112
-127.2942485607