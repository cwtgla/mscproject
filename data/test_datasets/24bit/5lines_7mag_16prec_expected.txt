127
5
211
15
8
174
0
7
152
230
16
145
255
11
126